packages/cdk8s-cli/templates/python-app/Pipfile

```diff
@@ -7,4 +7,4 @@ verify_ssl = true
 python_version = "3.7"
 
 [packages]
-constructs = "==2.0.2"
+constructs = "==4.1.0"
```

Comment:
@iliapolo Any updates on this PR? It would greatly help in my efforts to utilise `aws-cdk` and `cdk8s` in the same package. It seems that this version should be `3.0.4`, which I believe causes the integration tests to fail.

Reply:
@Rosenberg96 This it still planned for the near future, stay tuned.